### PR TITLE
Mb 5385 static analysis no console

### PR DESCRIPTION
**Pull request by kimallen**
## Description

This addresses the eslint rule of no-console.  The concern over console is that information could be revealed to a hacker that would give them an in to the application.  Initial findings include 12 occurrences. I removed one console.log as unnecessary and the rest have annotation. Three of the items include an error for which I determined there is more work to do for mitigation, which I will write a ticket for future work.

## Reviewer Notes

I determined the annotation based on the below:
From V-70273:
"If error messages are designed to provide users with just enough detail to pass along to support staff in order to aid in troubleshooting such as date, time, or other generic information, this is not a finding (Mitigated, CAT III).

If variable names, SQL strings, system path information, or source or program code are displayed in error messages sent to non-privileged users, this is a finding."

stories for future remediation:
https://dp3.atlassian.net/browse/MB-5597
https://dp3.atlassian.net/browse/MB-5595

## Setup

```yarn eslint```

## Code Review Verification Steps

* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-5384) for this change

### CODE REVIEW

File: src/registerServiceWorker.js [hunk 1]
```diff
@@ -22,57 +20,62 @@ const isLocalhost = Boolean(
...
-      }
-    });
-  }
-}
-
 function registerValidSW(swUrl) {
   navigator.serviceWorker
     .register(swUrl)
     .then((registration) => {
+      // eslint-disable-next-line no-param-reassign
```
Comment (lynzt) [inline src/registerServiceWorker.js:27]: assume we'll annotate this in a different pass...?

**Review by lynzt — COMMENTED** [review]
LGTM but holding off on approving as this PR is in DRAFT

File: src/registerServiceWorker.js (hunk 1 above)
Comment (jneuner-mitre) [reply src/registerServiceWorker.js:27]: I'm good with waiting.

**Review by jneuner-mitre — COMMENTED** [review]
lgtm for now, though I may revisit.  There's a pretty strong argument for all of these to be CAT III or even N/A

**Review by lynzt — APPROVED** [review]
LGTM and based on Jaron's ok, 🚢

**Review by suzubara — APPROVED** [review]

**Pull request merged**